src/components/recruting/_04_interview_notification/_01/PassList.tsx

```
@@ -0,0 +1,74 @@
+            <th className="py-[11px] w-[57px]">상태</th>
+            <th className="py-[11px] w-[100px]">이름</th>
+            <th className="py-[11px]">구분</th>
+            <th className="py-[11px]">순위</th>
+            <th className="py-[11px]">결과</th>
+          </tr>
+        </thead>
+        <tbody>
+          {filteredData.map((applicant) => (
+            <tr className="h-[50px] text-[12.85px] font-semibold border-b border-[#D6D7DA]">
+              <td>
+                <div className="bg-[#9EA2A9] rounded-[4.82px] text-white-100 flex-center py-[5px] px-[5px] text-[11px] font-semibold">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)